cm2350/e200_intc.py

```diff
@@ -6,13 +6,14 @@
 
 from envi.archs.ppc import regs as ppcregs
 
+from .ppc_peripherals import Module
 from .intc_const import *
 from . import intc_exc
 
 logger = logging.getLogger(__name__)
 
 
-class e200INTC:
+class e200INTC(Module):
```

Comment:
Is this supposed to be a peripheral?  Oh, this is the External INTC?  I wasn't thinking of that as a Device/Peripheral, but I guess it kinda is.

Reply:
since it behaves as a module and gets installed to `emu.modules` i assumed that this should be done